src/extension.ts

```diff
@@ -154,6 +155,21 @@ export async function activate (context: ExtensionContext): Promise<API | undefi
     // Activate
     commands.registerCommand('ethcode.activate', async () => {
       logger.success('Welcome to Ethcode!')
+    }),
+
+    // Load Foundry contracts
+    commands.registerCommand('ethcode.foundry.load', async () => {
+      try {
+        const isFoundry = await isFoundryProject()
+        if (isFoundry) {
+          await parseBatchCompiledJSON(context)
```

Comment:
I see you used `parseBatchCompiledJSON()` to load compiled contracts created by foundry. However I was expecting to use/improve ethcode contracts loading a bit.

Foundry, Remix, Hardhat or Solc can have different compiled output. And when we are creating separate commands like  `ethcode.foundry.load` then we might not want to use a general loading technique.

I think it will be more useful if we use a command like `parseFoundryCompiledJSON` and use specific code only to load foundry compiled contracts, it will be more modular in long term. Also if it can not load foundry configuration or can not load compiled files it should emit appropriate error messages.